WordPress/Classes/ViewRelated/Jetpack/Jetpack Restore/Restore Status/JetpackBackupStatusCoordinator.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+
+protocol JetpackBackupStatusView {
+    func render(_ backup: JetpackBackup)
+    func showError()
+    func showComplete()
+}
+
+class JetpackBackupStatusCoordinator {
+
+    private let service: JetpackBackupService
+    private let site: JetpackSiteRef
+    private let view: JetpackBackupStatusView
+
+    init(site: JetpackSiteRef,
```

Comment:
Just a heads up, I had issues using the `JetpackSiteRef` for the scanning view because I later needed more metadata that the `Blog` entity contains. I recently switched to using `Blog` instead of `JetpackSiteRef`. 

Not sure if you'll later need metadata such as site name, timezone, etc.

Reply:
Ah, good to know! Will switch to `Blog` instead if I end up needing that metadata. Thanks for the heads up! ❤️